AppCenterReactNativeShared/prepare-local-podspec.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Build AppCenterReactNativeShared.framework
+(cd $SRCROOT; $SRCROOT/build-fat-framework.sh)
+
+# Generate AppCenter-SDK-ReactNative-iOS-Pod-{version}.zip
+podspecFile=local.podspec $SRCROOT/../zip-framework.sh
```

Comment:
Can you explain this line?

Reply:
Passing `local.podspec` to `zip-framework.sh` script as I explained in https://github.com/microsoft/appcenter-sdk-react-native/pull/634/files#r306022447.